src/main/java/Katleen.java

```diff
@@ -0,0 +1,207 @@
+                   String[] splitSaved = save[0].split(" ");
+                   Task saved = getTask(savedTask, splitSaved[0], splitSaved, false);
+                   tasks.add(saved);
+                   if (!save[1].isEmpty()) {
+                       tasks.get(temp).setDone(true);
+                   }
+                   temp++;
+               }
+               }
+       } catch (IncompleteTaskException e) {
+           System.out.println("Could not create new task");
+       } catch (IOException e) {
+           System.out.println("IOException");
+       }
+   }
+
+    public static void main(String[] args) throws IncompleteTaskException, UnrecognizedCommandException {
+        System.out.println(LINE);
+        System.out.println("Hello! I'm Katleen.");
+        System.out.println("What can I do for you?");
+        System.out.println(LINE);
+        loadFileContents();
+        Scanner in = new Scanner(System.in);
+        String text = "";
+        while (!text.equals("bye")) {
+            text = in.nextLine();
+            System.out.println(LINE);
+            parseString(text);
+            System.out.println(LINE);
+        }
+    }
+
+    private static void parseString(String input) {
+        String[] splitInput = input.split(" ");
+        String cmdWord = splitInput[0];
+        int taskCount = tasks.size();
+        try {
+            switch (cmdWord) {
+            case "list":
+                if (tasks.isEmpty()) {
+                    System.out.println("You have no tasks!");
+                    break;
+                }
+                System.out.println("Here are the tasks in your list:");
+                int count = 1;
+                for (Task t : tasks) {
+                    System.out.print(count + ". ");
+                    t.printTask();
+                    count++;
+                }
+                break;
+            case "bye":
+                for (Task t : tasks) {
+                    FileWriter fw = new FileWriter(filename);
+                    fw.write(t.toString());
+                }
+                System.out.println(LINE);
+                System.out.println("Bye, have a nice day!");
+                System.out.println(LINE);
+                break;
+            case "mark":
+            case "unmark":
+            case "delete":
+                String index = splitInput[1];
+                int i = Integer.parseInt(index) - 1;
+                if (cmdWord.equals("delete")) {
+                    Task temp = tasks.get(i);
+                    tasks.remove(i);
+                    System.out.println("Task deleted: " + temp);
+                    break;
+                }
+                if (cmdWord.equals("mark")) {
+                    tasks.get(i).setDone(true);
+                    break;
+                }
+                tasks.get(i).setDone(false);
+                break;
+            case "todo":
+            case "deadline":
```

Comment:
add //Fallthrough here as well, do check other areas of your code for the same error